lms/static/js/fixtures/courseware/course_home_events.html

```diff
@@ -1,3 +1,15 @@
+<div class="upgrade-banner layout-1q3q layout-reversed">
```

Comment:
Can you update this fixture file so it's in line with the actual HTML? I don't think it is far off, but a few classes have changed.